### PR TITLE
Subject page updates

### PR DESCRIPTION
**Pull request by jenhadfield-dfe**
### Trello card
https://trello.com/c/MEasakGW/6231-create-subject-pages-template

### CODE REVIEW

File: app/views/content/is-teaching-right-for-me/maths/_article.html.erb [hunk 1]
```diff
@@ -1,91 +1,66 @@
...
-        }
-      ],
-      colors: %w[yellow],
-      numbered: false
-    ) %>
   </section>
-
   <section class="col col-space-m col-720">
-    <h2>Check your qualifications</h2>
+    <h2 class="heading--box-blue">Start your teaching journey</h2>
```
Comment (mariosh-e) [inline app/views/content/is-teaching-right-for-me/maths/_article.html.erb:62]:
```suggestion
    <h2 class="heading--box-blue">Start your journey into teaching</h2>
```

**Comment by mariosh-e** [comment]
Footer on physics page seems to odd on this page!

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 2]
```diff
@@ -1,96 +1,112 @@
...
+      text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
+      name: "Shiulee, Curriculum lead for computing and business",
+      large: true
+    ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll be teaching</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum for computing</a>, with opportunities to develop your pupils' computational thinking.</p>
+    <h4>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</h4>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/computing/_article.html.erb:26]: Need to change this to an h3 - you are skipping a heading level at the moment!

```suggestion
    <h3>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</h3>
```

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 3]
```diff
@@ -1,96 +1,112 @@
...
+      <li>Computer systems</li>
+      <li>Uses of technology</li>
+      <li>Online safety </li>
+    </ul>
+    <p>When you're teaching key stage 3 you could use interactive exercises during your lessons, like creating a robot car.</p>
+    <p><a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks3-l/units">Explore what teaching a key stage 3 computing lesson would be like</a>.</p>
+  </section>
+
+  <section class="col col-720">
+    <h4>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) computer science as a GCSE qualification include:</h4>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/computing/_article.html.erb:41]:
```suggestion
    <h3>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) computer science as a GCSE qualification include:</h3>
```

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 4]
```diff
@@ -1,96 +1,112 @@
...
-<li>a parent or carer</li>
-<li>disabled</li>
-</ul>
-<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
-<p>Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
-<h2>Paid internship </h2>
-<p>If you’re currently doing an undergraduate or master's degree and are interested in teaching computing, you could apply for a <a href="/is-teaching-right-for-me/teaching-internship-providers">paid teaching internship</a>.</p>
-<p>The 3-week programme could help you to understand what it’s really like in the classroom and get a feel for school life.</p>
-</section>
+    <h4>Themes you'll cover when you teach 16 to 18 year olds (key stage 5) computer science as an A level qualification include:</h4>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/computing/_article.html.erb:55]:
```suggestion
    <h3>Themes you'll cover when you teach 16 to 18 year olds (key stage 5) computer science as an A level qualification include:</h3>
```

**Review by gemmadallmandfe — COMMENTED** [review]
Some more skipped headings that need addressing @mariosh-e !

File: app/views/content/is-teaching-right-for-me/physics.md [hunk 5]
```diff
@@ -1,5 +1,12 @@
 ---
 title: Become a physics teacher
+title_paragraph: |-
+  <p>
+  Teaching secondary physics is an exciting and worthwhile career. Schools need more specialist physics teachers, so it's a great choice of subject.</p>
+  <p>
+  You could inspire your pupils to consider rewarding professions, from tackling climate change to artificial intelligence (AI). 
+  <p>
+  Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee physics teachers.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/physics.md:9]:
```suggestion
  Tax-free bursaries of $bursaries_postgraduate_physics$ or scholarships of $scholarships_physics$ are available for eligible trainee physics teachers.</p>
```

File: app/views/content/is-teaching-right-for-me/maths.md [hunk 6]
```diff
@@ -1,5 +1,12 @@
 ---
 title: Become a maths teacher
+title_paragraph: |-
+  <p>
+  If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.</p>
+  <p>
+  As a secondary maths teacher, an exciting career spent passing your knowledge on to the next generation could be for you. 
+  <p>
+  Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee maths teachers.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/maths.md:9]:
```suggestion
  Tax-free bursaries of $bursaries_postgraduate_maths$ or scholarships of $scholarships_maths$ are available for eligible trainee maths teachers.</p>
```

**Review by gemmadallmandfe — COMMENTED** [review]
@mariosh-e I've let some comments for you - all very minor - looks good!

If you are able to look at them this morning, we could get this out this afternoon?

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 7]
```diff
@@ -1,106 +1,112 @@
...
+      text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
+      name: "Shiulee, Curriculum lead for computing and business",
+      large: true
+    ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll be teaching</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum for computing</a>, with opportunities to develop your pupils' computational thinking.</p>
+    <h3>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</h3>
```
[the remark this answers is not in the document]
Comment (mariosh-e) [reply app/views/content/is-teaching-right-for-me/computing/_article.html.erb:26]: These don't need to be in H3 will standardise to make it look clearer - checked with @Sarah-DfE and she has agreed

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 8]
```diff
@@ -1,106 +1,112 @@
...
+    <ul>
+      <li>Algorithms</li>
```
[the remark this answers is not in the document]
Comment (mariosh-e) [reply app/views/content/is-teaching-right-for-me/computing/_article.html.erb:28]: @Sarah-DfE and I had this discussion yesterday so I will change to lowercase

File: app/views/content/is-teaching-right-for-me/computing/_article.html.erb [hunk 9]
```diff
@@ -1,106 +1,120 @@
...
+      <li>systems architecture</li>
+      <li>computing related mathematics</li>
+    </ul>
+    <p>When you're teaching key stage 5 you'll help pupils to develop websites and prepare them for further learning or employment.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+      text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that’s it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+      name: "Chris Farr, Teacher Training Adviser",
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/computing/_article.html.erb:70]:
```suggestion
      name: "Chris Farr, teacher training adviser",
```

Comment (gemmadallmandfe) [reply app/views/content/is-teaching-right-for-me/computing/_article.html.erb:70]: (we don't use capitals on job roles)

File: app/views/content/is-teaching-right-for-me/physics/_article.html.erb [hunk 10]
```diff
@@ -1,108 +1,109 @@
...
+      <li>mechanics and mechanical properties of matter</li>
+      <li>electric circuits and waves</li>
+      <li>matter, quantum and nuclear physics</li>
+    </ul>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+      text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that’s it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+      name: "Chris Farr, Teacher Training Adviser",
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/physics/_article.html.erb:53]:
```suggestion
      name: "Chris Farr, teacher training adviser",
```

File: app/views/content/is-teaching-right-for-me/engineers-teach-physics.md [hunk 11]
```diff
@@ -1,8 +1,17 @@
 ---
 title: Engineers teach physics
+title_paragraph: |-
+  <p>
+  Physics initial teacher training courses for engineers and material scientists.
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/engineers-teach-physics.md:5]: If you're an engineer or material scientist, an engineers teach physics initial teacher training course could be a great start to your teaching journey.

Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/engineers-teach-physics.md:5]:
```suggestion
  If you're an engineer or material scientist, an engineers teach physics initial teacher training course could be a great start to your teaching journey.
```

File: app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb [hunk 12]
```diff
@@ -101,28 +57,11 @@
...
-    </p>
-
-    <p>
-      <a href="/funding-and-support/scholarships-and-bursaries">Find out about your eligibility for bursaries and scholarships</a>.
-    </p>
-
-    <p>
-      You can also <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">search for physics teacher training courses</a> that are not part of the Engineers teach physics training programme.
+    <h3>Fund your teacher training</h3>
+    <p>Scholarships of <%= v :scholarships_physics %> or bursaries of <%= v :bursaries_postgraduate_physics %> are available for trainee physics teachers starting their teacher training course between September 2024 and July 2025.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb:62]:
```suggestion
    <p>Scholarships of <%= v :scholarships_physics %> or bursaries of <%= v :bursaries_postgraduate_physics %> are available for eligible trainee physics teachers.</p>
```

I suggest we remove the dates in line with other subject pages, to make future maintenance easier.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged on Monday morning

**Pull request merged**